evm/vm/forks/byzantium/opcodes.py

```diff
@@ -0,0 +1,126 @@
+import copy
+import functools
+
+from cytoolz import merge
+
+from evm import constants
+from evm.exceptions import (
+    WriteProtection,
+)
+from evm import opcode_values
+from evm import mnemonics
+
+from evm.opcode import as_opcode
+
+from evm.logic import (
+    call,
+    context,
+    logging,
+    storage,
+    system,
+)
+
+from evm.vm.forks.spurious_dragon.opcodes import SPURIOUS_DRAGON_OPCODES
+
+
+def no_static(opcode_fn):
```

Comment:
The name here had me confused at first, but then I realized it's used to ensure a given opcode_fn is not passed a computation with a static msg, right? Maybe rename it to `ensure_no_static()`?